Dockerfile.local

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8.13-bullseye
+
+RUN pip install --upgrade pip
+COPY ./requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . /tvp
+WORKDIR /tvp
+
+RUN apt-get update && apt-get -y install libgdal-dev
```

Comment:
so these gdallies can be found also within this image. Noice. Since afaik this was the main blocker for using python images... Ubuntu would've also an option but me likes this the best.